include/ppx/grfx/grfx_command.h

```diff
@@ -582,6 +582,16 @@ class CommandBuffer
     //
     void Draw(const grfx::FullscreenQuad* pQuad, uint32_t setCount, const grfx::DescriptorSet* const* ppSets);
 
+protected:
+    // Returns true when inside a render pass (dynamic or regular)
+    bool HasActiveRenderPass() const;
+
+    // Below values are for dynamic render passes only.
+    bool                                   mDynamicRenderPassActive = false;
+    grfx::Rect                             mRenderArea              = {};
+    std::vector<grfx::RenderTargetViewPtr> mRenderTargetViews       = {};
+    grfx::DepthStencilViewPtr              mDepthStencilView        = nullptr;
```

Comment:
perhaps we can either rename these or put them in a struct like `DynamicRenderPassInfo mCurrentDynamicRenderPassInfo` to make it clear that these are valid for an active dynamic render pass only